src/main.js

```diff
@@ -52,10 +51,7 @@ function getFunctionData(func) {
 
 function formatParams(params, module, funcCallerParam) {
     function formatError(err) {
-        return {
-            message: err.message,
-            stack: err.stack,
-        };
+        return err;
```

Comment:
Since we are just returning the error without any treatment by now, what do you think to remove this function, refactoring the places where it is called to use only the parameter instead of the function call?

Reply:
Done.